charts/istio/istio-ingress/values.yaml

```diff
@@ -1,6 +1,5 @@
 labels:
   app: istio-ingressgateway
-  istio: ingressgateway
```

Comment:
Out of curiosity, why was this label removed everywhere?

Reply:
The reason is that the labels for the istio resources are merged.
The ExposureClass handler ingress gateways would then have the default labels + additional labels. That means that  for Shoots w/o an ExposureClass configured their controlplane could be selected to receive traffic from an ingress gateway of an ExposureClass handler and then would never receive traffic. 
I rather tend to remove the labels entirely from the charts and default the labels for the default ingress gateway properly. That would be cleaner in my opinion.